pkg/restapi/restapi.go

```diff
@@ -27,6 +28,13 @@ func WithWebhookURLs(webhookURLs ...string) Opt {
 	}
 }
 
+// WithLabel is an option allowing for the label to be set.
+func WithLabel(label string) Opt {
```

Comment:
To me this is a default label.
I assume we will eventually extend the REST API to allow for overriding the default.

Reply:
Done!

Reply:
Btw, arcpy REST API doesn’t allow to override labels. They use ‘alias’ field for that purpose and alias is separate  field in request/response models & persisted records  .